reviews/ethblock.art/Testing.md

```diff
@@ -1,15 +1,44 @@
 # Testing
-## Done by:
+## Commit-SHA: [eeda149192144921c4ebbd74efa6d1f2c4e85cd8](https://github.com/adrianleb/blockart-contracts/commit/eeda149192144921c4ebbd74efa6d1f2c4e85cd8)
+## Done by: [mariuspod](https://github.com/mariuspod)
 ### Reviewed by:
 
-
-### Initial observations
-* Tests only check for reverts
+## Summary
+* There are only 4 tests in total and 2 of them are not passing.
```

Comment:
We could suggest/write some tests to expand the test coverage

Reply:
actually I'm doing this under `Recommendations`, would you put it into the `Summary` as well ?

Reply:
Sure, will do.